content/boilerplates/trace-generation.md

```diff
@@ -0,0 +1,8 @@
+The number of requests you need to send in order to see trace data depends on the sampling rate set at the time Istio was installed.
```

Comment:
Please avoid using boilerplates without considering the content around it. You are using numbered steps in the document but not in the boilerplate. Not only that, but the numbering of lists in the boilerplate won't sync with the numbering in the document if you do. Boilerplate needs to be used carefully and for fully independent content. 
In this case, I would add the steps to the flow in the appropriate place on each file as part of the respective numbered list.

Reply:
The text needs to be changed to fit into the numbered steps of the procedure. be explicit about what the command does to avoid confusing readers.

```suggestion
1. To see trace data, you must send requests to your service. The number of requests depends on Istio's sampling rate. You set this rate when you install Istio. The default sampling rate is 1%. You need to send at least 100 requests before the first trace is visible. To send a 100 requests to the `productpage` service, use the following command:
```

Reply:
I explicitly did not number this boilerplate, as I have thoughts about using it elsewhere (in a FAQ) and because it is a sub-step of the larger list in which it sits.

I have adopted your wording.